MISCELLANEOUS.md

```diff
@@ -1,3 +1,68 @@
 # Miscellaneous
 
 If things don't go in [roles](/ROLES.md) or [channels](/CHANNELS.md), I guess they must go in here.
+
+## Emojis
+
+### Static
+Name | Picture
+--- | ---
+:PepeKMS: | ![](https://cdn.discordapp.com/emojis/450062460074852362.png?v=1)
+:eggplanthand: | ![](https://cdn.discordapp.com/emojis/450062477393395712.png?v=1)
+:whopinged: | ![](https://cdn.discordapp.com/emojis/450062492014739457.png?v=1)
+:blobhmm: | ![](https://cdn.discordapp.com/emojis/450063616876609546.png?v=1)
+:blobhyperthink: | ![](https://cdn.discordapp.com/emojis/450063618306867200.png?v=1)
+:blobthonk: | ![](https://cdn.discordapp.com/emojis/450063618902458415.png?v=1)
+:reeeee: | ![](https://cdn.discordapp.com/emojis/450063621037228044.png?v=1)
+:hypers: | ![](https://cdn.discordapp.com/emojis/450063621385355265.png?v=1)
+:god: | ![](https://cdn.discordapp.com/emojis/450068990799380501.png?v=1)
+:OhRlyNow: | ![](https://cdn.discordapp.com/emojis/450068991210160129.png?v=1)
+:dab: | ![](https://cdn.discordapp.com/emojis/450068991222874114.png?v=1)
+:KMS: | ![](https://cdn.discordapp.com/emojis/450068991252103168.png?v=1)
+:smirk~1: | ![](https://cdn.discordapp.com/emojis/450068991579258880.png?v=1)
+:rollsafe: | ![](https://cdn.discordapp.com/emojis/450068991990300672.png?v=1)
+:thot: | ![](https://cdn.discordapp.com/emojis/450068992204210195.png?v=1)
+:NoU: | ![](https://cdn.discordapp.com/emojis/450068992284164106.png?v=1)
+:ummmmm: | ![](https://cdn.discordapp.com/emojis/450068992522977293.png?v=1)
+:thinkharder: | ![](https://cdn.discordapp.com/emojis/450068992594411521.png?v=1)
+:thonkharder: | ![](https://cdn.discordapp.com/emojis/450068992762183690.png?v=1)

```

Comment:
This is the same as :quitethoughtful:, so one can be removed.
https://github.com/CyberDiscovery/meta/blob/443faefc40514be9bb36923683661265046aa153/MISCELLANEOUS.md#L35

Reply:
Correction: similar, but not the same:
:quitethoughtful: https://cdn.discordapp.com/emojis/450657080995741712.png?v=1
:thinkharder: https://cdn.discordapp.com/emojis/450068992594411521.png?v=1

Reply:
_I guess we can keep both if you really want, but it feels redundant_